api/Window.json

```diff
@@ -1785,15 +1785,11 @@
             },
             "firefox": {
               "version_added": true,
-              "notes": [
-                "Before version 62 this function returns <code>null</code> when called on a <code>Window</code> which has no presentation, such as an <code>iframe</code> with <code>display: none;</code>."
-              ]
+              "notes": "Note: In Firefox before version 62 this function returned <code>null</code> when called on a Window with no presentation, such as an iframe with <code>display: none;</code> set on it. This was since changed in <a href='https://bugzil.la/1467722'>bug 1467722</a> — it now returns a <code>CSSStyleDeclaration</code> object which has 0 for the <code>length</code> property and an empty string for every declaration. More work is being done to further improve this situation in <a href='https://bugzil.la/1471231'>bug 1471231</a>."
```

Comment:
The `Note: ` prefix looks redundant to me.

Reply:
Actually it could be split up into three notes, e.g.

```html
See <a href='https://bugzil.la/1471231'>bug 1471231</a> for aligning its behavior with the specification.
Since Firefox 62, it returns a <code>CSSStyleDeclaration</code> with a <code>length</code> of 0 and an empty string for every declaration. (see <a href='https://bugzil.la/1467722'>bug 1467722</a>).
Before Firefox 62, it returned <code>null</code> when called on a window with no presentation such as an iframe with <code>display: none;</code> set on it.

Reply:
You are right that "Note: " is redundant; removed!

I don't want to split it into three notes, as I think this will look too messy in the final rendered version. Instead, I've streamlined the text and cut it down a lot.

Thanks for the comments!